flight/Modules/Airspeed/airspeed.c

```diff
@@ -176,7 +176,7 @@ MODULE_INITCALL(AirspeedInitialize, AirspeedStart);
  */
 static void airspeedTask(void *parameters)
 {
-	AirspeedSettingsUpdatedCb(AirspeedSettingsHandle());
+	AirspeedSettingsUpdatedCb(NULL, NULL, NULL, 0);
```

Comment:
This ```NULL, NULL, NULL, 0``` paradigm is used so frequently I would suggest having a wrapper function. This will help readability and maintainability, since long lists of ```NULL```s and ```0```s can be difficult to digest and check for correctness.